t/141-ssl-c-api-password.t

```diff
@@ -0,0 +1,584 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
```

Comment:
I think it's fine to use `t/151-ssl-c-api-password.t`, so that we don't need to rename `t/141-luajit.t` to `t/151-luajit.t`.
The num order is not important here:)